deezer.py

```diff
@@ -16,48 +16,91 @@ def env_constructor(loader, node):
 
 yaml.SafeLoader.add_constructor("!env", env_constructor)
 
-if __name__ == '__main__':
-    with open("config.yml", 'r') as f:
+
+async def build_track_entry(builder, el):
```

Comment:
probably should be called `add_*_entry` instead of `build_*` as it adds it beyond of just building

Reply:
nvm

Reply:
but wait.. it should at least return it back to the main func

Reply:
but wait.. it should at least return those to the main funcs

Reply:
> but wait.. it should at least return those to the main funcs

fixed